src/app/login/login.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="flex center ">
```

Comment:
I see you have a center class that uses auto horizontal margins. That works, you can also center using flex box. Definitely worth trying out!